mtg/json.go

```diff
@@ -0,0 +1,38 @@
+package mtg
+
+import (
+	"encoding/json"
+)
+
+// RaritiesToCards maps card rarities to lists of Cards. This map is used for generating packs.
+// This map holds all cards that we can possibly generate. This should be regarded as a temporary
+// solution, since we probably don't want to keep tens of thousands of cards in memory.
+var RaritiesToCards = make(map[CardRarity][]*Card)
+
+// ReadCardData encodes JSON data from a byte array from a single set into a Set struct.
+// The Cards from the Set are stored in the RaritiesToCards global var.
+func ReadCardData(contents []byte) {
+	var s Set
+	_ = json.Unmarshal(contents, &s)
+	// TODO(flp): better error handling
+
+	getCardRarity := func(c *Card) CardRarity {
+		switch {
+		case c.Rarity == "Common":
+			return Common
+		case c.Rarity == "Uncommon":
+			return Uncommon
+		case c.Rarity == "Rare":
+			return Rare
+		case c.Rarity == "Mythic Rare":
+			return Mythic
+		}
+		return BasicLand
```

Comment:
Per our discussion in slack, it makes sense that a `BasicLand` has its own rarity because one shows up in every booster.  Makes sense!

You don't have to change it to be this way, but I think it's more idiomatic to it this way:

```
	switch {
		case c.Rarity == "Common":
			return Common
		case c.Rarity == "Uncommon":
			return Uncommon
		case c.Rarity == "Rare":
			return Rare
		case c.Rarity == "Mythic Rare":
			return Mythic
                default:
                        return BasicLand
		}
```